catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -862,7 +862,7 @@ private void trySleep(long millis) {
     try {
       Thread.sleep(millis);
     } catch (InterruptedException e) {
-      // Do nothing
+      Thread.currentThread().interrupt();
```

Comment:
❗️ Although we need to figure out how to propagate interruptions, this will now cause a problem as it this method is called from within `tryUpdate` which is recursive and propagating here means the next attempts will also fail for the same reasons. We need to figure out how to retain the fact that we were interrupted and propagate the interruptions outside the whole thing.